client/src/ingredients/interactive-example.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export function InteractiveExample({ document }) {
+  return (
+    <div>
+      <iframe
+        title={`Interactive example for ${document.title}`}
+        className="interactive tabbed-standard"
```

Comment:
Do we need these? The `interactive` one seems useful for when we write a stylesheet. But what about `tabbed-standard`?

Reply:
Yeah, `tabbed-standard` is used to indicate how tall to make the iframe (some interactive examples are bigger than others). We should remove it (and I will do that in this PR) but we should also extend the interactive example grammar to let an author specify the height directly. And that part I'd like to do in a follow up.

Reply:
-> https://github.com/mdn/stumptown-experiment/issues/46